packages/cli/src/tools/config/index.ts

```diff
@@ -156,13 +143,7 @@ function loadConfig(projectRoot: string = findProjectRoot()): Config {
 
     const isPlatform = Object.keys(config.platforms).length > 0;
 
-    /**
-     * Legacy `rnpm` config required `haste` to be defined. With new config,
-     * we do it automatically.
-     *
-     * @todo: Remove this once `rnpm` config is deprecated and all major RN libs are converted.
-     */
-    const haste = config.haste || {
+    const haste = {
       providesModuleNodeModules: isPlatform ? [dependencyName] : [],
       platforms: Object.keys(config.platforms),
```

Comment:
I think we don't need to provide haste anymore since 0.61?

Reply:
Writing it down and will send a follow-up PR to remove it.